docs/Tutorials/Migrating-from-Orion-to-Tessera.md

```diff
@@ -193,3 +198,9 @@ You must specify the following options in order to run the migration tool:
 `password`= Target Tessera database password
 
 `url`= Target Tessera database JDBC connection string
+
+# Support
```

Comment:
Maybe be clear about the support type so we don't have to refuse requests from non customers who will be disappointed.
```suggestion
# Enterprise support
```